bpf/kprobe_pwru.c

```diff
@@ -59,6 +60,8 @@ struct event_t {
 	u64 addr;
 	u64 skb_addr;
 	u64 ts;
+	u32 cpu;
+	u32 pad;
```

Comment:
Did you check with `pahole` that this padding is needed?

Reply:
Great, `pahole` is the very tool that I need.